server/server/Controllers/customerCrud.js

```diff
@@ -0,0 +1,64 @@
+const mongoose = require("mongoose");
+const User = require("../Models/user");
+
+const getCustomerById = async (req, res) => {
+  const { id } = req.params;
```

Comment:
we will have user Id inside `req.user`
```suggestion
  const id = req.user.userId;
```